installer/pkg/config/v1/config.go

```diff
@@ -134,6 +135,10 @@ type ObjectStorageCloudStorage struct {
 	Project        string    `json:"project" validate:"required"`
 }
 
+type ObjectStorageAzure struct {
+	Certificate ObjectRef `json:"certificate"`
```

Comment:
```suggestion
	Credentials ObjectRef `json:"credentials"`
```

Reply:
Ok. I'll change the S3 one in that PR as well